test/testchunkingng.cpp

```diff
@@ -140,7 +141,7 @@ private slots:
         fakeFolder.setServerOverride([&](QNetworkAccessManager::Operation op, const QNetworkRequest &request, QIODevice *) -> QNetworkReply * {
             if (op == QNetworkAccessManager::PutOperation) {
                 // Test that we properly resuming, not resending the first chunk
-                Q_ASSERT(request.rawHeader("OC-Chunk-Offset").toLongLong() >= firstChunk.contentSize);
+                Q_ASSERT(request.rawHeader("OC-Chunk-Offset").toULongLong() >= firstChunk.contentSize);
```

Comment:
Are we sure this value is sent unsigned by the server?

Reply:
I'd hope so... I could assert it if needed...

Reply:
The header is set by our code and its a `qint64` https://github.com/owncloud/client/blob/e93cda468efd09ffe5f0b36f4cb1d27a57f586bf/src/libsync/propagateupload.h#L343-L343

Reply:
https://github.com/owncloud/client/pull/10129

Reply:
rebased on top of that PR, so now it's gone